dotCMS/src/main/resources/postgres.sql

```diff
@@ -560,6 +560,8 @@ create table contentlet_version_info (
    locked_by varchar(100),
    locked_on timestamptz,
    version_ts timestamptz not null,
+   variant_id varchar(255),
```

Comment:
it seems you have the same column twice, is that ok?

Reply:
done https://github.com/dotCMS/core/pull/22868/commits/b8a61e219cbd169ac8faa56d1c828326886e2bdb